### ISSUE TITLE
Crash when pressed a menu button

### ISSUE THREAD

**Issue by candysidner** [always-on/always#282]
This may be related to work on the focus for button presses.
https://www.dropbox.com/s/oxnp22fr3isumbg/CrashAtButtonPress.test

**Comment by charlesrich** [comment]
I assume you mean the Client crashed, because all I see in log is a socket disconnection from client at the end.  Transferring to @lring to see if he can reproduce.

**Comment by candysidner** [comment]
I don't know what failed.  I just know it was a crash.  And I tried to get it to happen when I used menus after that but nothing I tried anywhere did it.  I don't know what kind of slightly different thing I did when I pressed it.  Or maybe it was just at some odd but important time.
c.
On Feb 10, 2014, at 1:10 PM, Charles Rich notifications@github.com wrote:

> I assume you mean the Client crashed, because all I see in log is a socket disconnection from client at the end. Transferring to @lring to see if he can reproduce.
> 
> —
> Reply to this email directly or view it on GitHub.

---

Candace L. Sidner, Ph.D.
Research Professor
Department of Computer Science
Worcester Polytechnic Institute
(508) 831-6637
http://candysidner.com
sidner@wpi.edu
